src/app/accounts/register.component.ts

```diff
@@ -16,11 +16,25 @@ import { RegisterComponent as BaseRegisterComponent } from 'jslib/angular/compon
     templateUrl: 'register.component.html',
 })
 export class RegisterComponent extends BaseRegisterComponent {
+    showTerms = true;
+    acceptPolicies: boolean = false;
+
```

Comment:
Would it not have been better to put this all in jslib base component?

Reply:
It was a passing thought for me and it is possible. The reason I didn't do it, is because it does not apply to all platforms using jslib. 

Can easily create a jslib PR with these tracking variables and the setting of `showTerms` in the class constructor. Would leave the submit override in each necessary platform, however.

Thoughts?

Reply:
Which platforms does this not apply to? I can't think of any now.

Reply:
CLI and Directory Connector

Reply:
Neither of those uses this component (CLI isn't angular and neither offers registration). I think we can move all logic into jslib base registration component.

Reply:
On it!